lib/camerapreview.dart

```diff
@@ -224,21 +226,28 @@ class CameraAwesomeState extends State<CameraAwesome> with WidgetsBindingObserve
     if (Platform.isAndroid) {
       _initImageStream();
     }
-    _initAndroidPhotoSize();
     _initPhotoSize();
+    _initPreviewSize();
     _initVideoSize();
     camerasAvailableSizes = await CamerawesomePlugin.getSizes();
-    if (widget.selectDefaultSize != null) {
-      widget.photoSize.value = widget.selectDefaultSize!(camerasAvailableSizes);
+    if (widget.selectPreviewSize != null) {
+      widget.previewSize.value = widget.selectPreviewSize!(camerasAvailableSizes);
     } else {
-      widget.photoSize.value = camerasAvailableSizes[0];
+      widget.previewSize.value = camerasAvailableSizes[0];
     }
 
-    if (widget.selectDefaultVideoSize != null) {
-      widget.videoSize.value = widget.selectDefaultVideoSize!(camerasAvailableSizes);
+    if (widget.selectVideoSize != null) {
+      widget.videoSize.value = widget.selectVideoSize!(camerasAvailableSizes);
     } else {
       widget.videoSize.value = camerasAvailableSizes[0];
     }
```

Comment:
Just an FYI, this can be rewritten to:
```suggestion
    widget.videoSize.value = widget.selectVideoSize?.call(camerasAvailableSizes) ?? camerasAvailableSizes[0];
```

Reply:
👍